website/source/docs/acl/acl-auth-methods.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "docs"
+page_title: "ACL Auth Methods"
+sidebar_current: "docs-acl-auth-methods"
+description: |-
+  An Auth Method is a component in Consul that performs authentication against a trusted external party to authorize the creation of an appropriately scoped ACL token usable within the local datacenter.
+---
+
+-> **1.5.0+:**  This guide only applies in Consul versions 1.5.0 and newer.
+
+# ACL Auth Methods
+
+An Auth Method is a component in Consul that performs authentication against a
+trusted external party to authorize the creation of an appropriately scoped ACL
+token usable within the local datacenter.
+
+The only supported type of auth method in Consul 1.5 is
+[`kubernetes`](/docs/acl/auth-methods/kubernetes.html).
+
+## Overview
+
+Without auth methods a trusted operator needs to be critically involved in the
+creation and secure introduction of each ACL token to every application that
+needs one, while ensuring that the policies assigned to these tokens follow the
+principle of least-privilege.
+
+When running in environments such as a public cloud or when supervised by a
+cluster scheduler, applications may already have access to uniquely identifying
+credentials that were delivered securely by the platform. Consul auth method
+integrations allow for these credentials to be used to create ACL tokens with
+properly-scoped policies without additional operator intervention.
+
+In Consul 1.5.0 the focus is around simplifying the creation of tokens with the
+privileges necessary to participate in a [Connect](/docs/connect/index.html)
+service mesh with minimal operator intervention.
+
+## Operator Configuration
+
+An operator needs to configure each auth method that is to be trusted by
+using the API or command line before they can be used by applications.
+
+* **Authentication** - One or more **auth methods** should be configured with
+  details about how to authenticate application credentials. Successful
+  validation of application credentials will return a set of trusted identity
```

Comment:
kait: "trusted identity attributes such as ..., ..., and ..." I think a few examples would be helpful here or we could define them somewhere.